pycaret/time_series/forecasting/oop.py

```diff
@@ -3458,9 +3499,26 @@ def plot_model(
 
 
         display_format: str, default = None
```

Comment:
Please check out the functional.py file as well. It has the functional equivalent API to the OOP one and the docstring and arguments need to be updated there as well.

Reply:
should be resolved in [2dca48d](https://github.com/pycaret/pycaret/pull/2671/commits/2dca48d5816b31ffc91927688650a494b452efc3)